src/test/java/ru/training/at/hw1/CalculatorAddOperationTests.java

```diff
@@ -0,0 +1,29 @@
+package ru.training.at.hw1;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculatorAddOperationTests extends  BaseCalculatorTests {
```

Comment:
You should use Ctrl + Alt + L for autoformat style (Intellij IDEA) or don't use two spaces after "extends"